### PR TITLE
mp rest client IT

### PR DESCRIPTION
**Pull request by ajm01**
pr for the new mp rest client test bucket - i probably should have named the branch something else, sorry

### CODE REVIEW

File: proposals/boost/boost-maven/boost-maven-plugin/src/it/test-mpRestClient/invoker.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# User port 9000 to match mpConfig property
```
Comment (scottkurz) [inline proposals/boost/boost-maven/boost-maven-plugin/src/it/test-mpRestClient/invoker.properties]: Not about this file in particular..just the whole PR...For the other technologies we have a version in the dir name..e.g. `test-mpRestClient-1.0`... should we here?

File: proposals/boost/boost-maven/boost-maven-plugin/src/it/test-mpRestClient/pom.xml [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+			</plugin>
+		</plugins>
+	</build>
+
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>boost.boosters</groupId>
+				<artifactId>ee8-bom</artifactId>
+				<version>0.1.3-SNAPSHOT</version>
```
Comment (scottkurz) [inline proposals/boost/boost-maven/boost-maven-plugin/src/it/test-mpRestClient/pom.xml]: This BOM is imported in the mp20 one, right?   Not that you need to change it.

File: proposals/boost/boost-maven/boost-maven-plugin/src/it/test-mpRestClient/pom.xml [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+								<include>**/it/**/*.java</include>
+							</includes>
+							<!-- tag::system-props[] -->
+							<systemPropertyVariables>
+								<liberty.test.port>${testServerHttpPort}
+								</liberty.test.port>
+							</systemPropertyVariables>
+							<!-- end::system-props[] -->
+							<runOrder>reversealphabetical</runOrder>
+						</configuration>
```
Comment (scottkurz) [inline proposals/boost/boost-maven/boost-maven-plugin/src/it/test-mpRestClient/pom.xml]: We could make a bit more noise in the comments if there is an inter-test ordering dependency, which I think there is right?  E.g. here and in the `.java` parts with `@Test`.

File: proposals/boost/boost-maven/boost-maven-plugin/src/it/test-mpRestClient/src/main/java/io/openliberty/guides/inventory/InventoryManager.java [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+
+    private List<SystemData> systems = Collections.synchronizedList(new ArrayList<>());
+
+    private static String newline = System.getProperty("line.separator");
+
+    Logger LOG = Logger.getLogger(InventoryManager.class.getName());
+
+    @PersistenceContext(unitName = "guideapp-persister")
+    private EntityManager em;
+
```
Comment (scottkurz) [inline proposals/boost/boost-maven/boost-maven-plugin/src/it/test-mpRestClient/src/main/java/io/openliberty/guides/inventory/InventoryManager.java]: Is the persistence validated by the test logic?   (I didn't try to read the code, just figured I'd ask.)

Comment (ajm01) [reply proposals/boost/boost-maven/boost-maven-plugin/src/it/test-mpRestClient/src/main/java/io/openliberty/guides/inventory/InventoryManager.java]: removed all jpa/jdbc logic

File: proposals/boost/boost-maven/boost-maven-plugin/src/it/test-mpRestClient/invoker.properties (hunk 1 above)
Comment (ajm01) [reply proposals/boost/boost-maven/boost-maven-plugin/src/it/test-mpRestClient/invoker.properties]: added a version number to dir name

**Review by scottkurz — APPROVED** [review]

**Pull request merged**